phpBB/phpbb/event/dispatcher.php

```diff
@@ -40,4 +46,33 @@ public function trigger_event($eventName, $data = array())
 		$this->dispatch($eventName, $event);
 		return $event->get_data_filtered(array_keys($data));
```

Comment:
yeah, we just unnecessarily create the event\data object, filter it and so on, instead of just returning.
But I guess it's fine